test/gtest/ucp/ucp_test.h

```diff
@@ -242,6 +245,8 @@ class ucp_test : public ucp_test_base,
     void short_progress_loop(int worker_index = 0) const;
     void flush_ep(const entity &e, int worker_index = 0, int ep_index = 0);
     void flush_worker(const entity &e, int worker_index = 0);
+    void flush_worker(const entity &e, const std::vector<entity*> &entities,
```

Comment:
can we define
```cpp
void flush_worker(const entity &e, int worker_index = 0, 
                  const std::vector<entity*> &entities = entities());
```

and same for request_wait

Reply:
Fixed, cannot use non-static class members in default argument, but added `&entities = {}` instead. Now using `entities()` when the passed vector size is zero. (could not get rid of the two `check_events()` call as we are using a reference).